### ISSUE TITLE
jx version fails to return a version

### ISSUE THREAD

**Issue by rawlingsj** [jenkins-x/jx#138]
```
jx version
Server Version: ""
```

**Comment by cyrille-leclerc** [comment]
Still not there 

```
$ jx version
Error: unknown command "version" for "jx"
Run 'jx --help' for usage.

md5sum jx 
b73ab773a89a97042669430bbebacae3

sha1sum
8c749fc140fe0e87b9466ebd7a1b183808b3f3f3
```

**Comment by cyrille-leclerc** [comment]
@rawlingsj says it is fixed

**Comment by jstrachan** [comment]
should be fixed from 1.0.43 onwards
